src/wkwebkit.cs

```diff
@@ -749,32 +748,51 @@ interface WKWebView
 		[Export ("handlesURLScheme:")]
 		bool HandlesUrlScheme (string urlScheme);
 
-		[Mac (10, 15), iOS (13, 0)]
+		// TODO
+		// 11.4 beta2:
+		// WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
```

Comment:
there's a newer beta of the OS (but not a newer Xcode). Have you tried with a beta 3 device (or macOS) ?